phat/contracts/action_offchain_rollup/lib.rs

```diff
@@ -512,7 +512,8 @@ mod action_offchain_rollup {
                 ))))
                 .returns::<brick_profile::Result<H160>>()
                 .invoke()
-                .map_err(|_| Error::BadBrickProfile)?;
+                .log_err("failed to get evm address from profile")
```

Comment:
This would use the original pink::error which won't print the poll_id. Better print it it with the logging::error.

Reply:
Or let me implement anther log_err

Reply:
> Or let me implement anther log_err

Done in https://github.com/Phala-Network/phat-bricks/pull/31